oauth_manager.go

```diff
@@ -509,16 +513,42 @@ func (r *RedisOsinStorageInterface) GetClientTokens(id string) ([]OAuthClientTok
 		go r.store.RemoveSortedSetRange(key, "-inf", cleanupStartScore)
 	}
 
+	itemsPerPage := 2
```

Comment:
Not sure of how this will fit in with the config system. But I have it here right now so I could add tests and all of that.

If I am guided on how this will be added to the config or how it should be specified, I'd happily rework this PR

Reply:
Ended up going with 
https://github.com/TykTechnologies/tyk/pull/2109/commits/e3d35f385c4cac231e739958344553574a93fbb2 ..